modules/cli-installing-cli.adoc

```diff
@@ -60,7 +60,7 @@ You can install the OpenShift CLI (`oc`) binary on Linux by using the following
 .Procedure
 
 ifdef::openshift-origin[]
-. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ and choose the folder for your operating system and architecture.
```

Comment:
Although x86_64 is appropriate for Linux, the rest of this step text ("and choose the folder for your operating system and architecture.") doesn't match what follows after you've navigated to this new location. Because you've already chosen the operating system and architecture with this path.

And in this new location, there are a ton of options now, and I'm not positive, but it might be best to point them in the right direction of where to go next (or take them directly into one of the version subfolders).